CSS/index.css

```
@@ -0,0 +1,531 @@
+    position: absolute;
+    bottom: 0;
+    justify-content: center;
+}
+
+.main-ad-content{
+    gap: 32px;
+    display: flex;
+    flex-direction: column;
+}
+
+.main-ad-link{
+    text-decoration: none;
+}
+
+.main-ad-head{
+    font-family: 'Pretendard';
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 56px;
+}
+
+.button.main-ad-button{
+    border-radius: 40px;
+    text-align: center;
+    font-family: 'Pretendard';
+    font-size: 20px;
+    font-weight: 600;
+    line-height: 32px;
+    height: 56px;
+    padding: 16px 124px;
+    white-space: nowrap;
+    display: flex;
+    align-items: center;
+    cursor: pointer;
+}
+
+.main-ad-image{
+    max-width: 746px;
+    max-height: 340px;
+}
+
+.box{
+    width: 100%;
+    height: 720px;
+    
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.content{
+    display: flex;
+    justify-content: space-between;
+    border-radius: 12px;
+    width: 988px;
+    background-color: #FCFCFC;
+    white-space: nowrap;
+    margin: 0 auto;
+}
+
+.box-image{
+    width: 579px;
+    height: 444px;
+}
+
+.main-first-texts{
+    padding-right: 24px;
+    margin-top: 103px;
+    margin-bottom: 103px;
+}
+
+.main-second-texts{
+    padding-left: 24px;
+    display: flex;
+    flex-direction: column;
+    margin-top: 103px;
+    margin-bottom: 103px;
+    margin-right: 64px;
+    text-align: right;
+}
+
+.main-third-texts{
+    padding-right: 24px;
+    margin-top: 103px;
+    margin-bottom: 103px;
+    margin-left: 22px;
+}
+
+.section-head{
+    font-size: 18px;
+    font-weight: 700;
+    line-height: 26px;
+    font-family: "Pretendard";
+    color: #3692FF;
+    margin-bottom: 12px;
+}
+
+.bigtext{
+    font-family: "Pretendard";
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 56px;
+    letter-spacing: 2%;
+    color: #374151;
+    margin-bottom: 24px;
+}
+
+.smalltext{
+    font-family: "Pretendard";
+    font-size: 24px;
+    font-weight: 500;
+    line-height: 32px;
+    color: #374151;
+}
+
+.sub-ad{
+    width: 100%;
+    height: 540px;
+    background-color: #CFE5FF;
+    position: relative
+    
+}
+
+.sub-ad-box{
+    width: 100vw;
+    gap: 7px;
+    display: flex;
+    position: absolute;
+    bottom: 0;
+    justify-content: center;
+}
+
+.sub-ad-content{
+    padding: 0px 0px 60px 0px;
+    gap: 10px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    white-space: nowrap;
+    margin-right: 69px;
+    margin-top: 112.5px;
+}
+
+.sub-ad-head{
+    font-family: 'Pretendard';
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 56px;
+    color: #374151;
+}
+
+.sub-ad-image{
+    width: 746px;
+    height: 397px;
+}
+
+.footer{
+    background-color: #111827;
+    width: 100%;
+    height: 160px;
+    display: flex;
+    justify-content: center;
+}
+
+.footer-box{
+    max-width: 1120px;
+    width: 100%;
+    margin-left: 400px;
+    margin-right: 400px;
+    margin-top: 32px;
```

단축 속성으로 작성하면 깔끔하겠네요!
```suggestion
    margin: 32px 400px 0;
```